.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: New feature/Improvement of existing features
+about: For general improvements and the request of new functionalities
+title: "[FEAT] short summary of new feature/improvement"
+labels: enhancement
+assignees: ''
+
+---
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Detailed Description
+<!--- Provide a detailed description of the change or addition you are proposing -->
+
+## Context
+<!--- Why is this change important to you? How would you use it? -->
+<!--- How can it benefit other users? -->
+
+## Possible Implementation
+<!--- Not obligatory, but suggest an idea for implementing addition or change -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Version used:
+* Environment name and version (e.g. Django 3.2 on nginx 1.9.1):
+* Server type and version:
+* Operating System and version:
```

Comment:
Why environment info is needed on feature request?

Reply:
Bad copy paste, I'll fix in the next commit.